### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by Stephanie-Shen324** [valeoai/SLidR#12]
Hi,

Thank you for your previous replies. I have reproduced your results with your given minkunet checkpoint. Now I am trying to pretrain my own minkunet checkpoints. 

I want to ask what should the training loss be around at the end of pretraining?

**Comment by CSautier** [comment]
At the end of the pretraining, for the provided weights with minkunet, the training loss was ~2.7 and the validation loss around 1 points higher.
The PPKT training only reached ~3.5 training loss as the objective is harder.

**Comment by Stephanie-Shen324** [comment]
Thank you! I can confirm that the pretraining results are reproducible!
